README.md

```diff
@@ -4,19 +4,29 @@
 
 # Django Standardized Image Field
 
-Django Field that implement the following features:
+## Why would I want this?
 
-* Django-Storages compatible (S3)
-* Resize images to different sizes
+This is a drop-in replacement for the [Django ImageField](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ImageField) that provides a standardized way to handle image uploads.
```

Comment:
```suggestion
This is a drop-in replacement for the Django's [ImageField](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ImageField) that provides a standardized way to handle image uploads.
```

Reply:
'for the Django's'... shouldn't have 'the' I believe..

Reply:
Oh yes, certainly. Good spot.